octobot_trading/enums.py

```diff
@@ -407,6 +431,7 @@ class ExchangeConstantsPositionColumns(enum.Enum):
     MAINTENANCE_MARGIN_RATE = "maintenance_margin_rate"
     STATUS = "status"
     SIDE = "side"
+    ORIGINAL_SIDE = "original_side"
```

Comment:
As my first comment, I don't understand why we should keep the original side

Reply:
I removed it thats for the positions parser